main/SS/Formula/Eval/FunctionEval.cs

```diff
@@ -391,7 +391,7 @@ private static Function[] ProduceFunctions()
             retval[306] = new NotImplementedFunction("CHITEST"); // CHITEST
             retval[307] = new NotImplementedFunction("CORREL"); // CORREL
             retval[308] = new NotImplementedFunction("COVAR"); // COVAR
-            retval[309] = new NotImplementedFunction("FORECAST"); // FORECAST
+            retval[309] = new Forecast(); // FORECAST
```

Comment:
Ideally this should also support FORECAST.LINEAR, but I wasn't sure how to go about getting the ID of that function.

Reply:
You can reference how Apache POI implement this. Usually, it will create a static instance of the function and call it here

Reply:
It looks like they don't support FORECAST.LINEAR either.

https://github.com/apache/poi/blob/c960ee60709a4059796a13eec3b383ea4363f4fc/poi/src/main/java/org/apache/poi/ss/formula/eval/FunctionEval.java#L292

But I'll adjust to calling a static instance instead.